sites/kit.svelte.dev/src/lib/server/docs/types.d.ts

```diff
@@ -0,0 +1,23 @@
+export type DocsData = Category[];
+
+export interface Section {
+	title: string;
+	slug: string;
+	// Currently, we are only going with 2 level headings, so this will be undefined. In future, we may want to support 3 levels, in which case this will be a list of sections
```

Comment:
might be helpful to use a JSDoc-style comment so it appears in VS Code hovers
```suggestion
	/** Currently, we are only going with 2 level headings, so this will be undefined. In future, we may want to support 3 levels, in which case this will be a list of sections */
```